packages/core/types-tests-global-theme/tests.tsx

```diff
@@ -1,5 +1,6 @@
 // TODO: Test if Global implements default theme
```

Comment:
I think these types-tests-global-theme/tests.tsx can all go?

Especially if emotion-theming gets rolled in. As there will always be a theme and there will be no special themed variants.

Reply:
Yeah, seems to be the case. I thought of these tests additionally if the old API would stand parallel.